docs/content/latest/api/ysql/syntax_resources/commands/alter_user,alter_role_option,role_specification,alter_user_rename,new_role_name,alter_user_config,config_setting,config_param,config_value.grammar.md

```diff
@@ -0,0 +1,38 @@
+```
+alter_user ::= ALTER USER role_specification 
+               [ [ WITH ] alter_role_option [ , ... ] ]
+
+alter_role_option ::= SUPERUSER
+                      | NOSUPERUSER
+                      | CREATEDB
+                      | NOCREATEDB
+                      | CREATEROLE
+                      | NOCREATEROLE
+                      | INHERIT
+                      | NOINHERIT
+                      | LOGIN
+                      | NOLOGIN
+                      | CONNECTION LIMIT connlimit
+                      | [ ENCRYPTED ] PASSWORD  ' password ' 
+                      | PASSWORD NULL
+                      | VALID UNTIL  ' timestamp ' 
+
+role_specification ::= role_name | CURRENT_USER | SESSION_USER
+
+alter_user_rename ::= ALTER USER role_name RENAME TO new_role_name
+
+new_role_name ::= '<Text Literal>'
+
+alter_user_config ::= ALTER USER { role_specification | ALL } 
+                      [ IN DATABASE database_name ] config_setting
+
+config_setting ::= SET config_param { TO | = } 
+                   { config_value | DEFAULT }
+                   | SET config_param FROM CURRENT
+                   | RESET config_param
+                   | RESET ALL
+
+config_param ::= '<Text Literal>'
+
+config_value ::= '<Text Literal>'
```

Comment:
nit: I prefer keeping these "Text literal" d=rules out of the grammar and just use a followup "where config_value, ... "  are text literals.